torchvision/prototype/features/_feature.py

```diff
@@ -77,6 +77,13 @@ def __torch_function__(
         with DisableTorchFunction():
             output = func(*args, **kwargs)
 
+        # The __torch_function__ protocol will invoke this method on all types involved in the computation by walking
+        # the MRO upwards. For example, `torch.Tensor(...).to(features.Image(...))` will invoke
+        # `features.Image.__torch_function__` first. The check below makes sure that we do not try to wrap in such a
+        # case.
+        if not isinstance(args[0], cls):
+            return output
```

Comment:
Although the behavior should be kept for all ops where we might wrap, right now this only applies to `torch.Tensor.to`. `torch.Tensor.clone` does not accept any inputs and thus this method will only be invoked in case the data to clone is actually a feature. All other ops will be handled by the `else` branch which performs no wrapping.

Thus, we could also merge this into the `torch.Tensor.to` branch for now, but will need to put it back where it is right now as soon as we special case another op that takes tensors as inputs. I would keep it were it is, but won't oppose putting it elsewhere for now.